cxmon/utils/README.md

```diff
@@ -0,0 +1,22 @@
+# What
+suspend.bin is a MacBinary file which supposes to upack and run in M68k Macintosh only. It runs emul_op `0x7138` and trigger BasiliskII into cxmon so that you can add break points there.
```

Comment:
upack -> unpack

Reply:
Phrasing of "supposes to" doesn't make sense. Did you mean to write something like "which should be unpacked and run in ..."?

Reply:
Done!